ironfish/src/webApi.ts

```diff
@@ -178,6 +178,33 @@ export class WebApi {
     await axios.post(`${this.host}/telemetry`, payload)
   }
 
+  async createDeposit(payload: {
+    asset: string
+    source_address: string
+    destination_address: string
+  }): Promise<{ [keyof: string]: number }> {
+    this.requireToken()
+
+    const response = await axios.post<{ [keyof: string]: number }>(
+      `${this.host}/bridge/create`,
+      {
+        requests: [
+          {
+            ...payload,
+            source_chain: 'IRONFISH',
+            destination_chain: 'ETHEREUM',
+            source_transaction: null,
+            destination_transaction: null,
+            status: 'PENDING',
```

Comment:
I just updated the statuses to be `PENDING_PRETRANSFER`
https://github.com/iron-fish/ironfish-bridge/pull/5/files#diff-bc4eca7d21ff96af8c247ecbe052047e012a5f338813cc1f47f729ebc97d40bdL43

Reply:
changed this to `CREATED` -- I think that's right based on the flow in that PR